test/integration/targets/service/tasks/main.yml

```diff
@@ -1,6 +1,6 @@
 - name: skip unsupported distros
   meta: end_host
-  when: ansible_distribution in ['Alpine']
+  when: ansible_service_mgr not in ['rc', 'systemd', 'sysv', 'upstart']
```

Comment:
This incorrectly skips the tests on FreeBSD because `ansible_service_mgr` is `bsdinit`. Instead of including only systems listed here, it would be safer to exclude systems known not to be supported. That would avoid issues like this, where skipped tests could easily go undetected.

Reply:
Indeed. Good point. I've fixed the issue in the last commit.